python/packages/ai/teams/ai/prompts/message.py

```diff
@@ -26,13 +27,16 @@ class Message(Generic[T], DataClassJsonMixin):
         content (Optional[T]): Text of the message.
         function_call (Optional[FunctionCall]): A named function to call.
         name (Optional[str]): Name of the function that was called.
+        action_tool_calls (Optional[List[chat.ChatCompletionMessageToolCall]]):
+            List of tool calls from the model.
     """
 
     role: str
     content: Optional[T] = None
     context: Optional[MessageContext] = None
     function_call: Optional[FunctionCall] = None
     name: Optional[str] = None
+    action_tool_calls: Optional[List[chat.ChatCompletionMessageToolCall]] = None
```

Comment:
this is our message type so to keep things generic in types like this the name `action_calls` and mapping to our own internal type, not an openai type, helps us keep this code reusable with different clients/types/models.

Reply:
here's the interface for `ChatCompletionMessageToolCall`:
```ts
export interface ChatCompletionMessageToolCall {
  /**
   * The ID of the tool call.
   */
  id: string;

  /**
   * The function that the model called.
   */
  function: ChatCompletionMessageToolCall.Function;

  /**
   * The type of the tool. Currently, only `function` is supported.
   */
  type: 'function';
```
which imo doesn't map well enough with `FunctionCall`:
```ts
export interface FunctionCall {
    /**
     * Name of the function to call.
     */
    name?: string;

    /**
     * Optional. Arguments to pass to the function. Must be deserialized.
     */
    arguments?: string;
}
```
fThere's also `ChatCompletionAction` defined in our library, but that doesn't map well either. 
So I was thinking of making a new interface. Thoughts?
@aacebo @Stevenic

Reply:
I think it's best to create our own type, potentially called `ActionCall` that looks somewhat similiar to `ChatCompletionMessageToolCall`.

Reply:
yes, I agree